log/gps/gps_mgl_enigma_waypoint.ksy

```diff
@@ -0,0 +1,134 @@
+meta:
+  id: gps_mgl_enigma_waypoint
+  title: MGL Avionics Enigma Waypoint format
+  application: Enigma software
+  file-extension:
+    - ewd # WAYPOINT.EWD usually
+    - rte # MYROUTE.RTE usually
+  license: Unlicense
+  endian: le
+  encoding: ascii
+
+-license-header: | # the conditions contradict each other and seem to be unenforceable
+  MGL Avionics places this document and the data format it describes in the public domain in order to foster the use of a common, compact waypoint and route file format that is particularly optimized for use with resource limited systems and/or systems that require fast access to individual items in the waypoint files in order to increase overall systems performance.
+  In order to allow recognition of the format any party adopting this format agrees to refer to this format as the “Enigma waypoint format”. All data using this format should be considered to be in the public domain and the implementer will make reasonable effort to allow any interested party to use the data.
+  Exceptions:
+    MGL Avionics reserves copyright and denies use of this data format for any military or related purpose.
+
+doc: |
+  Waypoint format used by Enigma series of EFIS instruments from MGL Avionics.
+  It is common to create an index file containing a simple list of indexes, one to each entry in the waypoint file, sorted by a particular criteria (for example distance of the waypoint to a given location), rather than sorting the actual waypoint list itself. This is a method commonly employed by database applications.
+
+doc-ref:
+  - https://mglavionics.co.za/Docs/Enigma%20Waypoint%20format.pdf
+
+seq:
+  - id: waypoints
+    type: waypoint
+    repeat: eos
+    doc: There is no demand that waypoints are stored in any particular order in the waypoint file. They might be sorted by short name or waypoint type but this is of little consequence. The order of waypoints in the file matters only for route files. Route waypoints are stored in the file in order of their appearance in the route. A route should have at least one waypoint.
+
+types:
+  waypoint:
+    seq:
+      - id: lat
+        type: sph_coord
+        doc: N is positive
+      - id: lon
+        type: sph_coord
+        doc: E is positive
```

Comment:
Same as above, perhaps check using `valid`.

Reply:
(My comment is probably nonsense, see above: https://github.com/kaitai-io/kaitai_struct_formats/pull/381#discussion_r551049997)